### ISSUE TITLE
Compiling gives no ouptut but an error:getDiagnosticSeverity of undefined

### ISSUE THREAD

**Issue by pechaut78** [juanfranblanco/vscode-solidity#296]
Latest version of VSCode (1.62.0)
Latest version of vscode-solidity

Local or remote or node : press F5
Output:
Retrieving compiler information:
Compiler using solidity from node_module: solc solidity version: 0.8.9+commit.e5eed63a.Emscripten.clang

popup: Cannot read property getDiagnosticSeverity of undefined

**Comment by grimreaper619** [comment]
+1 
Have the same issue, started two days ago. Something I've noticed is this occurs only when there is compilation error (error in code). This also throws a different error 'Cannot read property uri of undefined'. Also the auto compilation doesn't track all syntax errors. Like, it will complain about missing semicolon, but doesn't show the incorrect variable name error.

**Comment by ejolanix** [comment]
+1
VS code version 1.62.0, Solidity v0.0.135 running in a WSL remote `Compiler using remote version: '0.8.9', solidity version: 0.8.9+commit.e5eed63a.Emscripten.clang`

And same as @grimreaper619, i.e. I only get popup `Cannot read property 'getDiagnosticSeverity' of undefined` if there is a compilation error, and no errors are listed in the Problems pane. The auto compilation feature doesn't work at all for me.

Also, I only get `Cannot read property uri of undefined` on the first solidity compile in a session.

There seems to be a problem parsing the errors output from the compile job.

**Comment by ejolanix** [comment]
Reverting to Solidity extension Solidity v0.0.134 removed the `Cannot read property 'getDiagnosticSeverity' of undefined` popup and compilation errors are now displayed in the Problems pane. The auto compilation feature also works again.

However, there is still a `Cannot read property uri of undefined` popup on the first solidity compile in a session.

**Comment by grimreaper619** [comment]
> Reverting to Solidity extension Solidity v0.0.134 removed the `Cannot read property 'getDiagnosticSeverity' of undefined` popup and compilation errors are now displayed in the Problems pane. The auto compilation feature also works again.

v0.0.136 seems to fix this issue. 

> However, there is still a `Cannot read property uri of undefined` popup on the first solidity compile in a session.

This is still happening, and auto compilation isn't working either. When clicking F5, it compiles succesfully and shows errors properly.

**Comment by grimreaper619** [comment]
As @ejolanix  mentioned, reverting to v0.0.134 removes that popup for me, but auto compilation is still broke. It was working perfectly before the upgrade, now it isn't even after the downgrade :(

**Comment by juanfranblanco** [comment]
Thanks for this, @grimreaper619 @ejolanix  @pechaut78  "Cannot read property uri of undefined" is the only issue you have at the moment?

**Comment by grimreaper619** [comment]
> Thanks for this, @grimreaper619 @ejolanix @pechaut78 "Cannot read property uri of undefined" is the only issue you have at the moment? Edit and F5 compilation..

Yes this is the only issue remaining. The other popup is gone as well as auto compilation is working now

**Comment by ejolanix** [comment]
From a limited amount of testing all issues seem to be resolved in v0.0.136 for me. If anything "pops up" I'll call out. Thanks @juanfranblanco for taking care of this 👍

**Comment by juanfranblanco** [comment]
Thanks @ejolanix  @grimreaper619
